department-of-veterans-affairs/vets-website
# Feature/20725 update focus and title handling around loading state of detail pages

## Description
Update the new appointment detail pages to make sure that browser is focusing on the header and loading indicators as the appointment is loaded and displayed on the detail pages. Also need to set an appropriate document title.

How to test:
under upcoming summary page, click on `details` link. Once the detailed page displays, however over the browser tab to see the document title

## Testing done


## Screenshots
**upcoming VA appointment**

![image](https://user-images.githubusercontent.com/54327023/110863338-8992d380-828e-11eb-8046-d1669818307e.png)
<hr />

**upcoming Community Care appt**
![image](https://user-images.githubusercontent.com/54327023/110863039-15583000-828e-11eb-8557-ad7feff63611.png)
<hr />

**Express Care Request:**
![image](https://user-images.githubusercontent.com/54327023/110863162-40428400-828e-11eb-9f23-6c78002f94e9.png)
<hr />

Canceled VA Request
![image](https://user-images.githubusercontent.com/54327023/111172247-0918ef00-857c-11eb-9859-710f51b904ec.png)


## Acceptance criteria
- [ ] For upcoming appointment (VA and CC) detail page and express care request page, document title should display the `appointment date` +  `type of care`
- [ ] For VA and CC request, the document title, should display either `Pending or Cancel` + `VA or Community care` + `type of care`

## Definition of done
